ngraph/core/include/ngraph/pattern/op/label.hpp

```diff
@@ -63,7 +63,8 @@ namespace ngraph
 
                 explicit Label(const element::Type& type = element::dynamic,
                                const PartialShape& s = PartialShape::dynamic())
-                    : Label(type, s, [](const Output<Node>&) { return true; }, OutputVector())
+                    : Label(
+                          type, s, [](const Output<Node>&) { return true; }, OutputVector())
```

Comment:
the old one was better IMO

Reply:
I didn't find style configuration flag to align this with the old style.